appveyor.yml

```diff
@@ -5,9 +5,8 @@ configuration: Release
 install:
     - ps: (new-object Net.WebClient).DownloadString("https://raw.github.com/madskristensen/ExtensionScripts/master/AppVeyor/vsix.ps1") | iex
 before_build:
-    - ps: Vsix-IncrementVsixVersion | Vsix-UpdateBuildVersion
+    - ps: Vsix-IncrementVsixVersion -versionType revision | Vsix-UpdateBuildVersion
     - ps: Vsix-TokenReplacement Conan.VisualStudio\source.extension.cs 'Version = "([0-9\\.]+)"' 'Version = "{version}"'
-    - ps: Vsix-TokenReplacement Conan.VisualStudio\source.extension.vsixmanifest '<Identity Id="VSConanPackage.4d0379e2-2698-4e66-89de-6ead71165e9f" Version="([0-9\\.]+)"' '<Identity Id="VSConanPackage.4d0379e2-2698-4e66-89de-6ead71165e9f" Version="{version}"'
```

Comment:
so, it's not needed any longer to patch vsixmanifest?

Reply:
No, it is already done by the IncrementVsixVersion, I didn't realize about that when I added this line 🤷‍♂